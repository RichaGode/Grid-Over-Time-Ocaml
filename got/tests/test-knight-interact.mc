int main()
{
  Knave k;
  Knight n; 
  k = new_knave(10,20);
  n = new_knight(10, 20);
  while (get_knight_health(n) > 0) {
    n = attack_knight(k, n);
    print(get_knight_health(n));
  }
  if (get_knight_health(n) == 0){
    knight_die(n);
  }

  return 0;

}
