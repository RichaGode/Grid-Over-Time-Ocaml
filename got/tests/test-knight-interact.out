75
50
25
0